presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -46,6 +46,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Function;
```

Comment:
This is only possible after 'Make unknown costs the default in PlanNodeCostEstimate.Builder' though. You didn't request this as a fixup, and if you did - all the commits in this PR would be fixups (just sayin'...)